nautobot/extras/datasources/git.py

```diff
@@ -411,6 +406,15 @@ def update_git_config_contexts(repository_record, job_result):
         "tenants",
         "tags",
     ):
+        if os.path.isdir(os.path.join(repository_record.filesystem_path, filter_type)):
+            job_result.log(
+                f'Found "{filter_type}" directory in the repository root. If this is meant to contain config contexts, '
+                "it should be moved into a `config_contexts/` subdirectory.",
+                level_choice=LogLevelChoices.LOG_WARNING,
```

Comment:
Would this be better suited to be an info instead of a warning? Thought being that the folder named "sites" isn't harmful to not be nested (maybe used for some other purpose) but may not have the desired affect.

Reply:
That's why it's a warning instead of a failure :-) IMHO info isn't a "sit up and take notice" log level, and I feel like it should be.